src/rulesets/rest/2022-05-25/json-api-rules/__tests__/resource-object-rules.test.ts

```diff
@@ -226,6 +226,8 @@ describe("resource object rules", () => {
       };
       const results = await ruleRunner.runRulesWithFacts(ruleInputs);
       expect(results.length).toBeGreaterThan(0);
+
+      console.log(results.filter((r) => !r.passed));
```

Comment:
Remove debugging console.log?

Reply:
Sure - there are some other test cases with these leftover (and they are quite helpful for debugging), but I can take them out 🤷‍♀️